src/libraries/System.Private.CoreLib/src/System/Globalization/CultureData.cs

```diff
@@ -1408,7 +1409,7 @@ internal string[] LongTimes
         }
 
         /// <summary>
-        /// short time format
+        /// (user can override) short time format
```

Comment:
>(user can override)  [](start = 12, length = 20)

please move this comment to its own line.

Reply:
This is how we have it in all properties where the user can override. Should we do the same for all those?

Reply:
It is confusing to have written this way. but it is not that important. I'll leave it to you :-)

---
In reply to: [445747823](https://github.com/dotnet/runtime/pull/38372#discussion_r445747823) [](ancestors = 445747823)